src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -136,6 +137,10 @@ export class ResultLayout extends Component {
     ResponsiveResultLayout.init(this.root, this, {});
   }
 
+  public get activeLayouts(): { [key: string]: IActiveLayouts } {
```

Comment:
This method is failing the build because it's returning an explicit interface `IActiveLayouts` but the interface is not exported from the module.

You'll need to export that interface.


From the build log : 
```
/home/travis/build/coveo/search-ui/src/ui/ResultLayout/ResultLayout.ts(140,48): error TS4043: Return type of public property getter from exported class has or is using private name 'IActiveLayouts'.
```